### PR TITLE
Ohpipeline 1.155.1133 cmake

### PR DESCRIPTION
**Pull request by DoomHammer**
### Summary
Changes to recipe:  **ohPipeline/1.155.1133**

#### Motivation
Version update

#### Details

---
- [X] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [X] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [X] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

File: recipes/ohpipeline/all/conanfile.py [hunk 1]
```diff
@@ -48,15 +48,15 @@ def build_requirements(self):
         self.tool_requires("cmake/[>3.23 <4]")
 
     def requirements(self):
-        self.requires("ohnet/1.36.5182", transitive_headers=True, transitive_libs=True)
+        self.requires("ohnet/[>=1.36.5182]", transitive_headers=True, transitive_libs=True)
```
[the remark this answers is not in the document]
Comment (DoomHammer) [reply recipes/ohpipeline/all/conanfile.py:51]: I wanted it to be usable with a wider range of versions as they are usually compatible with each other.

**Comment by DoomHammer** [comment]
@AbrilRBS seems to be good now :)

**Review by perseoGI — APPROVED** [review]
LGTM

**Review by AbrilRBS — APPROVED** [review]
Thanks! We can discuss the availability of ohnet as a version range in the future, we would just need to ensure it has compatibility guarantees

**Comment by DoomHammer** [comment]
Happy to learn more about it!

**Pull request merged**